### PR TITLE
Emil time report

### PR DESCRIPTION
**Pull request by Frequinzy**

### CODE REVIEW

File: report.md [hunk 1]
```diff
@@ -98,6 +113,9 @@ Anna
 - SEMAT section of report (30m)
 
 Emil
+- UML diagram (120m)
```
Comment (annamaartensson) [inline report.md]: Can you be a bit more specific here? We should have a granularity of 30-60m

File: report.md [hunk 2]
```diff
@@ -98,6 +113,9 @@ Anna
...
+- Documenting added library (60m)
+- Finalizing Pull Request (90m)
```
Comment (annamaartensson) [inline report.md]: Same thing here

**Review by annamaartensson — COMMENTED** [review]
See comments about granularity. Otherwise LGTM.

**Review by annamaartensson — APPROVED** [review]
LGTM

**Review by Tian0602 — APPROVED** [review]

**Pull request merged**